src/main/scala/com/databricks/labs/overwatch/utils/Tools.scala

```diff
@@ -137,10 +138,28 @@ object JsonUtils {
 
 }
 
+trait HelperTrait{
```

Comment:
we don't need a trait for this. Adding a trait to the object adds a lot of unnecessary complexity. We can discuss.